examples/schema_site_vrf_region/main.tf

```diff
@@ -31,8 +63,54 @@ resource "mso_schema_site_vrf_region" "vrfRegion" {
     subnet {
       ip    = "1.20.30.4"
       name  = "subnet1"
-      zone  = "us-east-1b"
       usage = "gateway"
     }
   }
 }
+
+// AWS specific example
+resource "mso_schema_site_vrf_region" "vrf_region_aws" {
+  schema_id          = mso_schema.demo_schema.id
+  template_name      = one(mso_schema.demo_schema.template).name
+  site_id            = data.mso_site.demo_site.id
```

Comment:
Would it be better to reference site_id here with mso_schema_site.demo_schema_site.id rather than data.mso_site.demo_site.id?

Reply:
ok